src/app/views/sidebar/resource-explorer/ResourceLink.tsx

```diff
@@ -54,12 +70,61 @@ const ResourceLink = (props: IResourceLinkProps) => {
 
   const items = getMenuItems();
 
-  return <span className={linkStyle.link}>
-    {!!resourceLink.iconresourceLink && <Icon style={{ margin: '0 4px' }}
-      {...resourceLink.iconresourceLink} />}
-    {resourceLink.name}
+  const setMaxOverflowWidth = () : string => {
+    const compensation = compensateForLevelPx();
+    if (windowWidth > 800 && windowWidth < 1024){ return `${130 - compensation}px`}
```

Comment:
We may need these window width values again when working on making some elements more responsive.
I think it would be better to save these in their own file.